src/aihwkit/inference/calibration/calibration.py

```diff
@@ -119,7 +119,9 @@ def calibrate_input_ranges(
 
     Only tiles that support and have enabled input range learning will
     be calibrated. If noise management is turned on an error is
-    raised.
+    raised.  Note: this implementation transiently registers a new `forward_pre_hook`
```

Comment:
Can you make a newline before "Note:" and after like:
```
[..] raised.

Note: 
      This implementation transiently registers a new `forward_pre_hook` ...
      [and so on with spaces at the beginning]
```
In this way the docstring will be interpreted as Note with SPHINX

Reply:
OK.  will do.